mxcube3/core/models/configmodels.py

```diff
@@ -35,13 +35,31 @@ class UIPropertiesListModel(BaseModel):
     __root__: Dict[str, UIPropertiesModel]
 
 
+class UserManagerUserConfigModel(BaseModel):
```

Comment:
Configuration model for user, mostly added for future use. Like this one can assign any role to a user through configuration (For site specific adaptation). This part is still not used by the UserManager